$MirOS: ports/lang/ruby/patches/patch-configure_in,v 1.1.7.1 2005/03/18 15:47:50 tg Exp $
--- configure.in.orig	Fri Sep  7 09:38:28 2007
+++ configure.in	Fri May 30 19:46:17 2008
@@ -1038,7 +1038,8 @@ if test "$with_dln_a_out" != yes; then
 			  test "$GCC" = yes && test "$rb_cv_prog_gnu_ld" = yes || LDSHARED="ld -Bshareable"
 			fi
 			rb_cv_dlopen=yes ;;
-	openbsd*) 	: ${LDSHARED="\$(CC) -shared ${CCDLFLAGS}"}
+	openbsd*|mirbsd*)
+			: ${LDSHARED="\$(CC) -shared ${CCDLFLAGS}"}
 			if test "$rb_cv_binary_elf" = yes; then
 			    LDFLAGS="$LDFLAGS -Wl,-E"
 			fi
@@ -1340,7 +1341,7 @@ if test "$enable_shared" = 'yes'; then
 	   LIBRUBY_ALIASES=""
 	fi
  	;;
-    openbsd*)
+    openbsd*|mirbsd*)
 	SOLIBS='$(LIBS)'
 	LIBRUBY_SO='lib$(RUBY_INSTALL_NAME).so.$(MAJOR).'`expr ${MINOR} \* 10 + ${TEENY}`
 	;;
